src/modules/RA/DataSource/components/DataSourceWMTSField.js

```diff
@@ -13,7 +13,7 @@ const WMTSFields = ({ translate: t, ...props }) => (
     <NumberInput source="minzoom" label="datasource.form.min-zoom" step={1} />
     <NumberInput source="maxzoom" label="datasource.form.max-zoom" step={1} />
     <NumberInput source="tile_size" label="datasource.form.tile-size" defaultValue={256} step={1} validate={required()} />
-    <TextInput source="url" type="text" label="datasource.form.wmts-url" validate={required()} />
+    <TextInput source="url" type="text" label="datasource.form.wmts-url" validate={required()} fullWidth />
```

Comment:
As you removed `type="text"` from another `<TextInput />`, shouldn't it be removed from here too?

```suggestion
    <TextInput source="url" label="datasource.form.wmts-url" validate={required()} fullWidth />
```

Reply:
Right !